changelog.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 - Multipart content delivery using websubhub-client.
```

Comment:
I think this is incorrect. We don't need add unplanned tasks under unreleased section. Unrelease section is basically there to say what to expect in the coming release. Also unreleased section should shave same structure as added, removed, changed etc.